<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CConfigFileSettingsBaseDialog</class>
 <widget class="QDialog" name="CConfigFileSettingsBaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration File Settings</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="Q3GroupBox" name="groupBox11">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_lineeditconfigfile"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Default configuration file</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="m_pushbuttonbrowse">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Q3GroupBox" name="groupBox12">
     <property name="title">
      <string>Configuration File loading</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="m_radiobuttonloadauto">
        <property name="text">
         <string>Automatically load configuration file upon start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="m_radiobuttonloadprompt">
        <property name="text">
         <string>Prompt for configuration file to load upon start</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="m_radiobuttonloadnoaction">
        <property name="text">
         <string>Take no action</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Q3GroupBox" name="m_radiobuttonnoaction">
     <property name="title">
      <string>Configuration File saving</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="0">
       <widget class="QRadioButton" name="m_radiobuttonsaveprompt">
        <property name="text">
         <string>Prompt for configuration file to save upon exit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="m_radiobuttonsavenoaction">
        <property name="text">
         <string>Take no action</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="m_radiobuttonsaveauto">
        <property name="text">
         <string>Automatically save configuration file upon exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="m_pushbuttonreset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>Abandon all changes</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttoncancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>Abandon all changes</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttonapply">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>Apply changes</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttonok">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Apply changes and close window</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_lineeditconfigfile</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>lineeditconfigfile_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonbrowse</sender>
   <signal>clicked()</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>pushbuttonbrowse_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radiobuttonloadauto</sender>
   <signal>clicked()</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>radiobuttonloadauto_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radiobuttonloadprompt</sender>
   <signal>clicked()</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>radiobuttonloadprompt_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radiobuttonloadnoaction</sender>
   <signal>clicked()</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>radiobuttonloadnoaction_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radiobuttonsaveauto</sender>
   <signal>clicked()</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>radiobuttonsaveauto_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radiobuttonsaveprompt</sender>
   <signal>clicked()</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>radiobuttonsaveprompt_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radiobuttonsavenoaction</sender>
   <signal>clicked()</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>radiobuttonsavenoaction_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttoncancel</sender>
   <signal>clicked()</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>pushbuttoncancel_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonapply</sender>
   <signal>clicked()</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>pushbuttonapply_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonok</sender>
   <signal>clicked()</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>pushbuttonok_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonreset</sender>
   <signal>clicked()</signal>
   <receiver>CConfigFileSettingsBaseDialog</receiver>
   <slot>pushbuttonreset_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
